services/metadactyl-clj/src/metadactyl/routes/tools.clj

```diff
@@ -23,260 +23,261 @@
         :query [params SecuredQueryParams]
         :return nil
         :summary "Delete Container Image"
         :description "Deletes a container image from the system."
-        (service/trap uri delete-image image-id)))
+        (ok (delete-image image-id))))
 
  (defroutes* data-containers
-   (GET* "/" [:as {uri :uri}]
+   (GET* "/" []
          :query [params SecuredQueryParams]
          :return DataContainers
          :summary "List Data Containers"
          :description "Lists all of the available data containers."
-         (service/trap uri list-data-containers))
+         (ok (list-data-containers)))
 
-   (GET* "/:data-container-id" [:as {uri :uri}]
+   (GET* "/:data-container-id" []
          :path-params [data-container-id :- DataContainerIdParam]
          :query [params SecuredQueryParams]
          :return DataContainer
          :summary "Data Container"
          :description "Returns a JSON description of a data container."
-         (service/trap uri data-container data-container-id)))
+         (ok (data-container data-container-id))))
 
 (defroutes* admin-data-containers
-  (PATCH* "/:data-container-id" [:as {uri :uri}]
+  (PATCH* "/:data-container-id" []
           :path-params [data-container-id :- DataContainerIdParam]
           :query [params SecuredQueryParams]
           :body [body DataContainerUpdateRequest]
           :return DataContainer
           :summary "Update Data Container"
           :description "Updates a data container's settings."
-          (service/trap uri modify-data-container data-container-id body)))
+          (ok (modify-data-container data-container-id body))))
 
 (defroutes* tools
-  (GET* "/" [:as {uri :uri}]
+  (GET* "/" []
         :query [params ToolSearchParams]
         :return ToolListing
         :summary "Search Tools"
         :description "This endpoint allows users to search for a tool with a name or description that
         contains the given search term."
-        (ce/trap uri #(search-tools params)))
+        (ok (search-tools params)))
 
-  (GET* "/:tool-id" [:as {uri :uri}]
+  (GET* "/:tool-id" []
         :path-params [tool-id :- ToolIdParam]
         :query [params SecuredQueryParams]
         :return Tool
         :summary "Get a Tool"
         :description "This endpoint returns the details for one tool."
-        (ce/trap uri #(get-tool tool-id)))
+        (ok (get-tool tool-id)))
 
-  (GET* "/:tool-id/container" [:as {uri :uri}]
+  (GET* "/:tool-id/container" []
         :path-params [tool-id :- ToolIdParam]
         :query [params SecuredQueryParams]
         :return ToolContainer
         :summary "Tool Container Information"
         :description "This endpoint returns container information associated with a tool. This endpoint
         returns a 404 if the tool is not run inside a container."
-        (ce/trap uri (requester tool-id (tool-container-info tool-id))))
+        (requester tool-id (tool-container-info tool-id)))
```

Comment:
The `requestor` macro takes care of that. However, there are other routes in here that need the `ok` still.